controllers/group_totals.go

```diff
@@ -36,19 +39,14 @@ func pipelineExecutor(rw http.ResponseWriter, r *http.Request, pp models.Pipelin
 	}
 }
 
-func ConditionTotalHandler(rw http.ResponseWriter, r *http.Request) {
-	pipelineExecutor(rw, r, models.NewConditionPipeline, "count")
-}
-
-func EncounterTotalHandler(rw http.ResponseWriter, r *http.Request) {
-	pipelineExecutor(rw, r, models.NewEncounterPipeline, "count")
-}
-
-func PatientListHandler(rw http.ResponseWriter, r *http.Request) {
```

Comment:
Oops!  We can't get rid of `PatientListHandler` -- the Population Filter on the Patients page needs it!  It calls `/GroupList/{id}`, which is routed to the `PatientListHandler`.  It looks like you'll need to implement this without the pipeline.